python/lsst/ip/isr/isrTask.py

```diff
@@ -836,6 +890,18 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
 
         detector = inputs['ccdExposure'].getDetector()
 
+        if self.config.doCrosstalk is True:
+            # Crosstalk sources need to be defined by the pipeline
+            # yaml if they exist.
```

Comment:
Perhaps add some kind of `else` catch for if doCrosstalk is requested but no sources are defined?

Reply:
I've added a log warning for the case where no crosstalkSources exist but there are interChip terms in the calibration.  The original code didn't raise anything for interChip coefficients that don't have matching sources, so I'm wary against breaking something by raising here, but you're right that it should say something so it's not a silent failure.